README.md

```diff
@@ -10,60 +10,17 @@ A minimalist SDK for accessing the [Docker Engine API](https://docs.docker.com/e
 - ```docker_buffer``` gets response data
 - ```docker_destroy``` destroys docker object and frees any associated memory
 
-The library is pretty low-level for now (i.e., doesn't support parsing ```JSON``` responses or have any functions higher-level than ```post``` or ```get```). An example application:
-
-```
-#include "docker.h"
-
-int main() {
-
-  fprintf(stderr, "curl_version: %s\n", curl_version());
-
-  DOCKER *docker = docker_init("v1.25");
-
-  if (docker) {
-    CURLcode response = docker_post(docker, "http://v1.25/containers/create",
-                                    "{\"Image\": \"alpine\", \"Cmd\": [\"echo\", \"hello world\"]}");
-    if (response == CURLE_OK) {
-      fprintf(stderr, "%s\n", docker_buffer(docker));
-    }
-
-    response = docker_get(docker, "http://v1.25/images/json");
+## Building the Code
+- Run the compile script to build the code. By default the example will be part of the build process. ( If anybody wants to not build the example,
+ they can manually edit the base CMakeLists. In future, the compile script will accomodate the option.)
+- If the build is successful, then a deb package will be created in the build directory. Use the command ```sudo dpkg -i <deb package name>```
+to install the package.
+- This will add libcdockerinterface to the /usr/lib directory and the example executable cdockerinterfaceexample in /usr/bin
```

Comment:
These names are mouthfuls; I understand where you're going with them, but can we keep the name or is there conceptual/actual namespace collision?

Reply:
No there is no collision.